apps/server/src/modules/user-login-migration/service/migration-check.service.ts

```diff
@@ -3,6 +3,7 @@ import { EntityId, LegacySchoolDo, UserDO, UserLoginMigrationDO } from '@shared/
 import { UserLoginMigrationRepo } from '@shared/repo';
 import { LegacySchoolService } from '@src/modules/legacy-school';
 import { UserService } from '@src/modules/user';
+import { SchoolInMigrationError } from '@src/modules/authentication/errors/school-in-migration.error';
```

Comment:
Do you really need the whole import? 
Until errors should be enough when the error itself will be exported in the barrel file